x11-libs/libxau/libxau-1.0.8.recipe

```diff
@@ -6,7 +6,7 @@ LibXau is used by both Xlib and XCB."
 HOMEPAGE="https://www.x.org/"
 COPYRIGHT="1988, 1993, 1994, 1998  The Open Group"
 LICENSE="MIT (no promotion)"
-REVISION="5"
+REVISION="7"
```

Comment:
Is that a bit too big for a jump?

Reply:
Seems I needed to update the local master for haikuports too, fix in a minute :)